src/test/java/SetTest.java

```diff
@@ -0,0 +1,51 @@
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+        numbers.add(4);
+    }
+
+    @Test
+    void test_requirement_1() {
+        int expecting = 4;
+        int result = numbers.size();
+
+        assertThat(result).isEqualTo(expecting);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(크기);
```

collection의 경우 위와 같이 구현도 가능함
이 테스트 성공 하나요?
크기가 4가 아닐텐데요.
한번 확인해 보면 좋겠네요

Reply:
제가 A처럼 했어야 했는데.. 요구사항을 잘못 적어넣어서 이런 문제가 발생한 것 같습니다😅

<img width="306" alt="image" src="https://github.com/next-step/java-racingcar/assets/59079053/285881e0-81b6-4156-9a3d-6de35daeae1a">
<img width="295" alt="image" src="https://github.com/next-step/java-racingcar/assets/59079053/ba413cf7-36df-46f5-8826-44699e4a3884">